libs/vm-modules/examples/commodity_prediction/script/commodity_prediction.etch

```diff
@@ -37,8 +37,12 @@ if (System.Argc() != 4)
   return;
 endif
 
-var loader = DataLoader();
-loader.addData("commodity", System.Argv(2), System.Argv(3));
+// LOAD THE DATA
```

Comment:
Does the script work now? ;)

Reply:
Not entirely - fixing it will require ML-383 so I'm going to leave it for now.